src/main/java/org/broadinstitute/hellbender/engine/GATKTool.java

```diff
@@ -337,7 +327,7 @@ private void validateSequenceDictionaries() {
 
         // Check all variants dictionaries against the reference and/or reads dictionaries
         // TODO: pass file names associated with each sequence dictionary into validateDictionaries(); issue #660
-        for ( SAMSequenceDictionary variantsDict : variantDicts ) {
+        for ( final SAMSequenceDictionary variantsDict : variantDicts ) {
```

Comment:
Comments and variable names need updating if we're actually potentially considering non-variant Feature dictionaries as well for validation purposes.